src/main/java/frc/robot/RobotContainer.java

```diff
@@ -94,13 +96,14 @@ public RobotContainer(RobotId robotId) {
 
         // Create Drivetrain
         m_drive = new Drivetrain(m_drivetrainTab, m_swerveModulesTab, m_vision);
-        m_driver = new GameControllerDriverConfig(m_drive, m_controllerTab, false);
+        m_operator = new Operator();
+
+        m_driver = new GameControllerDriverConfig(m_drive, m_operator, m_controllerTab, false);
```

Comment:
fairly minor but consider just giving the driver a supplier for the node. It's kinda weird the driver controller "has" the operator controller

Reply:
I originally had selectedNode as public so I wouldn't have to do that, but I had to change it to private. I think a supplier might make it even more confusing. (using m_operator.getSelectedNodeSupplier() instead of m_operator).